csharp_dotnetcore/12.NLP-With-LUIS/LuisBot.bot

```diff
@@ -0,0 +1,38 @@
+﻿{
+    "name": "Luis-Bot",
```

Comment:
How about just "LuisBot"? This "-" is a problem everywhere....

Reply:
Fixed